agent/config/config_test.go

```diff
@@ -269,6 +290,26 @@ func TestInvalidValueContainerStartTimeout(t *testing.T) {
 	assert.Equal(t, conf.ContainerStartTimeout, minimumContainerStartTimeout, "Wrong value for ContainerStartTimeout")
 }
 
+func TestZeroValueDockerPullInactivityTimeout(t *testing.T) {
+	defer setTestRegion()()
+	defer setTestEnv("ECS_DOCKER_PULL_INACTIVITY_TIMEOUT", "0s")()
+	ctrl := gomock.NewController(t)
+	mockEc2Metadata := mock_ec2.NewMockEC2MetadataClient(ctrl)
+	conf, err := NewConfig(mockEc2Metadata)
+	assert.NoError(t, err)
+	assert.Equal(t, conf.DockerPullInactivityTimeout, defaultDockerPullInactivityTimeout, "Wrong value for DockerPullInactivityTimeout")
+}
+
+func TestInvalidValueDockerPullInactivityTimeout(t *testing.T) {
```

Comment:
how is this different from the test `TestInvalidValueDockerInactivityTimeout`?

Reply:
It's a different input edge case test that follows the same setup / teardown and has the same result.  Its useful -- makes sure we don't accidentally handle the zero case incorrectly in the future.